ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -121,8 +121,12 @@ public NetworkMetrics(MetricRegistry registry) {
     selectorNodeMetricMap = new HashMap<String, SelectorNodeMetric>();
   }
 
-  public void initializeSelectorMetricsIfRequired(final AtomicLong activeConnections) {
-    selectorActiveConnections = new Gauge<Long>() {
+  /**
+   * Initializes a few network metrics for the selector
+   * @param activeConnections count of current active connections
+   */
+  void initializeSelectorMetrics(final AtomicLong activeConnections) {
+    numActiveConnections = new Gauge<Long>() {
```

Comment:
Haven't looked at the rest, but this `Gauge` is never registered as far as I can see.